src/Functions/readme.md

```diff
@@ -484,9 +504,6 @@ directive:
   - where:
       subject: (.*)StaticSite(.*)
     remove: true
-  - from: source-file-csharp
```

Comment:
Just FYI @Francisco-Gamino we've enabled customizing psd1 file via the "metadata" section in readme.md (see above green part), so we won't need to play with C# code here.

Reply:
Great, thanks @isra-fel for looping me in.